cmd/skopeo/inspect.go

```diff
@@ -135,9 +145,13 @@ func (opts *inspectOptions) run(args []string, stdout io.Writer) (retErr error)
 		}, opts.retryOpts); err != nil {
 			return errors.Wrapf(err, "Error reading OCI-formatted configuration data")
 		}
-		err = json.NewEncoder(stdout).Encode(config)
+		if len(opts.format) == 0 {
+			err = json.NewEncoder(stdout).Encode(config)
+		} else {
+			err = outputFormat(config, opts.format)
```

Comment:
Hum, output is supposed to go the `stdout` parameter, not to `os.Stdout` (to allow writing in-process unit tests), but the `formats` package does not support that.